dev-libs/zug/zug-0.1.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Transducers for C++"
+HOMEPAGE="https://sinusoid.es/zug/"
+SRC_URI="https://github.com/arximboldi/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Boost-1.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE=""
+#RESTRICT="!test? ( test )"
+SLOT=0
+
+RDEPEND="
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+"
```

Comment:
No deps at all, for sure? Then don't add these empty vars.

Reply:
Most of these libraries are very minimal and do not require any deps besides a C++ compiler.
I think that is a implicit dependency for ebuilds, right?

I will remove these, thanks